src/components/widgets/TreeView/TreeViewLeaf.js

```diff
@@ -39,16 +43,36 @@ const TreeViewLeaf = ({
           </em>
         </small>)
       </span>}
+    {isPublic &&
+      <OverlayTrigger
+        placement="bottom"
+        overlay={
+          <Tooltip id={`${id}-public-tooltip`}>
+            <FormattedMessage
+              id="app.groupTree.treeViewLeaf.publicTooltip"
+              defaultMessage="The group is public"
+            />
+          </Tooltip>
+        }
+      >
+        <Icon
+          name="eye"
+          className="text-muted"
+          style={{ marginLeft: '0.5em' }}
```

Comment:
You? Really?

Reply:
I know, but TreeView has no css/less file yet and this is just one tiny margin. Furthermore, we may refactor this once group flags are implemented.